recipes/tensorflow-lite/all/CMakeLists.txt

```diff
@@ -1,8 +1,37 @@
 cmake_minimum_required(VERSION 3.1)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup()
+include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(Eigen3 REQUIRED)
+find_package(absl REQUIRED)
+
+add_library(NEON_2_SSE INTERFACE IMPORTED)
+if (NOT CMAKE_SYSTEM_PROCESSOR OR CMAKE_SYSTEM_PROCESSOR MATCHES "x86")
+    set_property(TARGET NEON_2_SSE PROPERTY INTERFACE_LINK_LIBRARIES CONAN_PKG::intel-neon2sse)
+endif()
+
+add_library(farmhash INTERFACE IMPORTED)
+set_property(TARGET farmhash PROPERTY INTERFACE_LINK_LIBRARIES CONAN_PKG::farmhash)
```

Comment:
CONAN_PKG will be deprecated in the future, is not recommended anymore. Also, don't solve targets or more complex situation on wrapper, it should be very simple, just a wrapper. The main approach is using cmake, cmake_find_package generators, then the upstream cmake can use them without any addition .

Reply:
Thanks for the tips, just pushed an update for using `find_package` instead of `CONAN_PKG` as suggested.

> Also, don't solve targets or more complex situation on wrapper

Assuming you're referring to the check of `CMAKE_SYSTEM_PROCESSOR`.

In the [main CMakeLists.txt](https://github.com/tensorflow/tensorflow/blob/31a3e515a153ed2b646d0f11872acf802f66028d/tensorflow/lite/CMakeLists.txt#L502) from tensorflow the dependency `NEON_2_SSE` is required and included regardless of the architecture.

On the other hand, in the conan version of neon2sse [the use of the dependency is prevented](https://github.com/conan-io/conan-center-index/blob/08ebff8c0872d86c88fb14bcb405fbb9bbfd4b1c/recipes/intel-neon2sse/all/conanfile.py#L42) during validation.

I believe the behavior of conan's `intel-neon2see` is correct, as the library is only providing a wrapper of "arm_neon.h" for intel architectures. When building for arm, `tensorflow-lite` simply never includes the wrapper (but still requires the CMake dependency).

Do you have any suggestion on how to handle this differently from what I've done?